spec/services/meeting_starter_spec.rb

```diff
@@ -26,9 +26,10 @@
 
   describe '#call' do
     it 'calls BigBlueButtonApi with the right params' do
-      expect_any_instance_of(BigBlueButtonApi)
+      allow_any_instance_of(BigBlueButtonApi)
         .to receive(:start_meeting)
         .with(room:, options:, presentation_url:)
+        .and_return(meeting_starter_response)
```

Comment:
Making this change actually makes this test do nothing. `expect` and `allow` do 2 very different things

Reply:
It's a cop that made me change it.

`Prefer `allow_any_instance_of` over `expect_any_instance_of` when configuring a response.`

Reply:
I removed the return.